### PR TITLE
[I18N] Localization in Hindi

### LINKED ISSUE #912: [I18N] Localization in Hindi
It's noticeable that we have a very large community around us from India 🇮🇳 so I thought it would be a good idea to have in our app localization support for Hindi.

### PR DESCRIPTION
**Pull request by divyanshub024**
@RocketChat/android

Closes #912

### CODE REVIEW

File: app/src/main/res/values-hi-rIN/strings.xml [hunk 1]
```diff
@@ -0,0 +1,145 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (Shailesh351) [inline app/src/main/res/values-hi-rIN/strings.xml]: Remove this to edit translations in Translations editor

File: app/src/main/res/values-hi-rIN/strings.xml [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+    <string name="msg_content_description_log_in_using_google">Google द्वारा लॉगिन करें</string>
+    <string name="msg_content_description_log_in_using_linkedin">Linkedin द्वारा लॉगिन करें</string>
+    <string name="msg_content_description_log_in_using_meteor">Meteor द्वारा लॉगिन करें</string>
+    <string name="msg_content_description_log_in_using_twitter">Twitter द्वारा लॉगिन करें</string>
+    <string name="msg_content_description_log_in_using_gitlab">Gitlab द्वारा लॉगिन करें</string>
+    <string name="msg_content_description_send_message">मेसेज भेजें</string>
+    <string name="msg_content_description_show_attachment_options">अटैचमेंट विकल्प दिखाएं</string>
+    <string name="msg_you">आप</string>
+    <string name="msg_unknown">अनजान</string>
+    <string name="msg_email_address">ईमेल पता</string>
```
Comment (Shailesh351) [inline app/src/main/res/values-hi-rIN/strings.xml]: ई-मेल एड्रेस

File: app/src/main/res/values-hi-rIN/strings.xml [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+    <string name="message_user_left">ने चैनल को छोड़ दिया है</string>
+    <string name="message_user_joined_channel">चैनल में शामिल हो गया है</string>
+    <string name="message_welcome">%s का स्वागत करते हैं</string>
+    <string name="message_removed">संदेश हटाया गया</string>
+    <string name="message_pinned">एक संदेश पिन किया:</string>
+
+    <!-- Message actions -->
+    <string name="action_msg_reply">जवाब दें</string>
+    <string name="action_msg_edit">संपादन करें</string>
+    <string name="action_msg_copy">प्रतिलिपि</string>
```
Comment (Shailesh351) [inline app/src/main/res/values-hi-rIN/strings.xml]: May be "नकल" or "कापी" will be better

File: app/src/main/res/values-hi-rIN/strings.xml [hunk 4]
```diff
@@ -0,0 +1,145 @@
...
+    <string name="msg_username_or_email">यूजरनेम या ईमेल</string>
+    <string name="msg_password">पासवर्ड</string>
+    <string name="msg_name">नाम</string>
+    <string name="msg_email">ईमेल</string>
+    <string name="msg_avatar_url">अवतार यूआरएल</string>
+    <string name="msg_or_continue_using_social_accounts">या सामाजिक खाते का उपयोग करना जारी रखें</string>
+    <string name="msg_new_user">नया उपयोगकर्ता? %1$s</string>
+    <string name="msg_new_user_agreement">आगे बढ़कर आप हमारे %1$s और %2$s से सहमत हो रहे हैं</string>
+    <string name="msg_2fa_code">कोड 2FA</string>
+    <string name="msg_more_than_ninety_nine_unread_messages" translatable="false">99+</string>
```
Comment (Shailesh351) [inline app/src/main/res/values-hi-rIN/strings.xml]: if `translatable="false"` than no need to add it in other translations

File: app/src/main/res/values-hi-rIN/strings.xml [hunk 5]
```diff
@@ -0,0 +1,145 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="app_name">Rocket.Chat</string>
```
Comment (Shailesh351) [inline app/src/main/res/values-hi-rIN/strings.xml]: if `translatable="false"` than no need to add it in other translations

**Review by Shailesh351 — COMMENTED** [review]
Looks good except changes mentioned above.
Thanks

**Comment by rafaelks** [comment]
@divyanshub024 Could you please take a look on @Shailesh351 and let us know? Thank you!

File: app/src/main/res/values-hi-rIN/strings.xml (hunk 3 above)
Comment (divyanshub024) [reply app/src/main/res/values-hi-rIN/strings.xml]: I think "कॉपी" will be better than "नकल"

**Comment by divyanshub024** [comment]
Thanks @Shailesh351 for suggested changes. @rafaelks @leonardoaramaki all required changes are done.

**Review by Shailesh351 — APPROVED** [review]
LGTM

**Comment by leonardoaramaki** [comment]
Thanks a lot @divyanshub024 and @Shailesh351!

**Pull request merged**